index.js

```diff
@@ -70,127 +182,181 @@ function displayPrjct(title, description, backgroundUrl, idHtml) {
-  const h1SectionM = document.createElement('h2');
-  const UlSectionM = document.createElement('ul');
-  const li1SectionM = document.createElement('li');
-  const li2SectionM = document.createElement('li');
-  const li3SectionM = document.createElement('li');
-  const btn1SectionM = document.createElement('button');
-  const btn2SectionM = document.createElement('button');
-  const btn3SectionM = document.createElement('button');
-  const btnEnSectionM = document.createElement('button');
-
-  sectionMultiple.classList.add('section_M');
-  pSectionM.classList.add('p_sectionM');
-  h1SectionM.classList.add('h1_3');
-  UlSectionM.classList.add('ul_sectionM');
-  li1SectionM.classList.add('li_sectionM');
-  li2SectionM.classList.add('li_sectionM');
-  li3SectionM.classList.add('li_sectionM');
-  btn1SectionM.classList.add('butn_M');
-  btn2SectionM.classList.add('butn_M');
-  btn3SectionM.classList.add('butn_M');
-  btnEnSectionM.classList.add('btn_M_en');
-  sectionMultiple.setAttribute('id', `section_m${projectDisplay.idHtml}`);
-  UlSectionM.setAttribute('id', `ul_section_m${projectDisplay.idHtml}`);
-  pSectionM.setAttribute('id', `pSectionM${projectDisplay.idHtml}`);
-  h1SectionM.setAttribute('id', `h1SectionM${projectDisplay.idHtml}`);
-  btnEnSectionM.setAttribute('id', `btnEnSectionM${projectDisplay.idHtml}`);
+  const btnEnSectionMDesk = myTags[11];
+  const sectionMultiple = myTags[0];
+  sectionProjects.appendChild(myTags[0]);
+  myTags[0].append(myTags[2], myTags[1], myTags[3], myTags[10], myTags[11]);
+  myTags[3].append(myTags[4], myTags[5], myTags[6]);
+  myTags[4].appendChild(myTags[7]);
+  myTags[5].appendChild(myTags[8]);
+  myTags[6].appendChild(myTags[9]);
 
-  const sectionProjects = document.getElementById('Sections_M_Container');
+  myTags[0].style.background = `linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(${projectDisplay.backgroundUrl})`;
+  myTags[0].style.backgroundSize = '100% 100%';
 
-  const body = document.querySelector('body');
-  const section1 = document.getElementById('section1');
-  const MainSectionproject = document.getElementById('my_projects');
-  const sectionAbout = document.getElementById('section_aboutMe');
-  const contactForm = document.getElementById('contact_form');
-  const NavAll2 = document.getElementById('NavBar2');
-  const NavAll1 = document.getElementById('NavBar1');
+  function OpenPopUp() {
+    const MyPopUpTags = [
+      {
+        name: 'btnsList',
+        class: 'btns_List',
+        type: 'div',
+      },
+      {
+        name: 'closeImg',
+        class: 'close_popUp',
+        type: 'img',
+        src: 'images/HideNavIcon.png',
+      },
+      {
+        name: 'closePopUp',
+        class: 'btn1_popUp',
+        type: 'button',
+      },
+      {
+        name: 'btn1PopUp',
+        class: 'btn1_popUp',
+        type: 'button',
+      },
+      {
+        name: 'btn2PopUp',
+        class: 'btn1_popUp',
+        type: 'button',
```

Comment:
![image](https://user-images.githubusercontent.com/43564324/162262095-a159f16e-e759-482f-95f0-a159d5340312.png)

[optional] some sections of your code are not showing properly in the desktop version. it is important you fix this design problem so that users can easily navigate your modal popup page